solidity/apps/test/governance/governanceRouter.test.ts

```diff
@@ -426,6 +426,38 @@ describe('GovernanceRouter', async () => {
     });
   });
 
+  describe('Built-in checkpointing', () => {
```

Comment:
nit: consider moving these tests up to where the remote calls are made rather than a separate set of tests, e.g.

```
    it('governor can make remote calls', async () => {
      const value = 13;
      const call = formatCall(testSet, 'set', [value]);
      await router.callRemote(domains[1], [call]);
      await abacus.processMessages();
      expect(await testSet.get()).to.equal(value);
    });

    it('governor making remote calls result in a checkpoint', async () => {
    ...
    });
```

That said, these tests should really be testing `comboDispatch` in `router.test.ts`

Reply:
Yeah good idea, I was mostly following the interchain gas payment structure, but agree both should be pulled up.

I'm not sure what exactly you mean "be testing comboDispatch". There are tests in there that test `comboDispatch`, these tests are essentially just testing that `comboDispatch` is called correctly